interface/main/main_screen.php

```diff
@@ -116,6 +116,10 @@
 if ($GLOBALS['new_tabs_layout']) {
```

Comment:
Are you sure about that?  [PHP](http://php.net/manual/en/reserved.variables.globals.php) would apply this to all future script executions for all users (GLOBAL ly). Will need to check globals.php here but - 

If that library always resets $GLOBALS with database values (which would be poorly designed use of $GLOBALs), when this app makes next server call it will face original issue.  

If the library retains the setting, next user login without default frame layout selection (relying on global default) will get tabs layout on a random basis.